router/router.gno

```diff
@@ -46,19 +46,25 @@ func SwapRoute(
 
 	// check route length ( should be 1 ~ 7 )
 	routes := strings.Split(strRouteArr, ",")
-	require(1 <= len(routes) && len(routes) <= 7, ufmt.Sprintf("[ROUTER] router.gno__SwapRoute() || len(routes) should be 1 ~ 7 (len(routes):%d)", len(routes)))
+	if !(1 <= len(routes) && len(routes) <= 7) {
+		panic(ufmt.Sprintf("[ROUTER] router.gno__SwapRoute() || len(routes) should be 1 ~ 7 (len(routes)[%d])", len(routes)))
+	}
```

Comment:
If the conditional expression is long, I prefer to assign it to a variable. In this case, using a name like `isValidRouteLength` could improve readability.

```go
isValidRouteLength := 1 <= len(routes) && len(routes) <= 7
if !isValidRouteLength {
    // ...
}
```

Reply:
https://github.com/gnoswap-labs/gnoswap/blob/e26ff5d08b36b76aefcb3aaf98489f36e6d05eef/router/router.gno#L51-L54